Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -545,7 +545,15 @@ private func findPackageDirectory(
   containing path: AbsolutePath,
   _ fileSystem: FileSystem) -> AbsolutePath? {
   var path = path
-  while !fileSystem.isFile(path.appending(component: "Package.swift")) {
+  while true {
+    let packagePath = path.appending(component: "Package.swift")
+    if fileSystem.isFile(packagePath) {
+      let contents = try? fileSystem.readFileContents(packagePath)
+      if contents?.cString.contains("import PackageDescription") == true {
```

Comment:
Maybe we should just check for "PackageDescription" to avoid false negatives due to whitespace or comments?